docker/prognostic_run/Dockerfile

```diff
@@ -17,10 +17,20 @@ COPY external/fv3kube /fv3net/external/fv3kube
 COPY workflows/post_process_run /fv3net/workflows/post_process_run
 COPY workflows/prognostic_c48_run/ /fv3net/workflows/prognostic_c48_run
 COPY external/emulation /fv3net/external/emulation
-COPY projects/microphysics/scripts /fv3net/projects/microphysics/scripts
```

Comment:
This is not actually a package and was accidentally duplicated from a copy statement later on in the Dockerfile.